ext/cfx-ui/src/app/app.module.ts

```diff
@@ -49,7 +49,8 @@ const l10nConfig: L10nConfig = {
 	locale: {
 		languages: [
 			{ code: 'en', dir: 'ltr' },
-			{ code: 'fr', dir: 'ltr' }
+			{ code: 'fr', dir: 'ltr' },
+			{ code: 'br', dir: 'ltr' }
```

Comment:
This states the ISO code for this is BR, meanwhile in the ISO-639 definitions this is known as the Brenton language. I think (not 100% sure) that [pt](https://www.loc.gov/standards/iso639-2/php/langcodes_name.php?iso_639_1=pt) is more fitting.